gita/__main__.py

```diff
@@ -79,6 +80,14 @@ def f_add(args: argparse.Namespace):
             if new_groups:
                 print(f'Created {len(new_groups)} new group(s).')
                 utils.write_to_groups_file(new_groups, 'a+')
+        if new_repos and args.group:
+            gname = args.group
+            if gname in groups:
+                gname_repos = set(groups[gname]['repos'])
+                gname_repos.update(repos)
+                groups[gname]['repos'] = sorted(gname_repos)
+                print(f'Added repo to the {gname} group')
```

Comment:
also show the number of new repos added: `Added x new repo(s) to ...`

Reply:
done